components/front-page/Topics.tsx

```diff
@@ -0,0 +1,106 @@
+import Carousel from "components/ui/Carousel";
+import { CmsTopicHeader } from "lib/cms/topics";
+import { chunk, isObject, isString } from "lodash-es";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+export const Topics = ({
+  topics,
+  imagePlaceholders,
+  selectedTopic,
+  onClick,
+}: {
+  topics: CmsTopicHeader[];
+  imagePlaceholders: string[];
+  selectedTopic?: CmsTopicHeader | string;
+  onClick?: (topic: CmsTopicHeader) => void;
+}) => {
+  const router = useRouter();
+
+  return (
+    <>
+      <div className="hidden w-full gap-3 md:flex">
+        {topics.map((topic, index) => (
+          <div
+            key={index}
+            className={`
+            ztg-transition flex flex-1 cursor-pointer items-center gap-4 rounded-lg p-3 transition-all md:max-w-sm md:hover:scale-[1.015]
+              ${
+                (isString(selectedTopic) && selectedTopic === topic.slug) ||
+                (isObject(selectedTopic) && selectedTopic.slug === topic.slug)
+                  ? "bg-gray-200"
+                  : "bg-white"
+              }
+            `}
+            onClick={() => {
```

Comment:
Could you use `next/link` instead to retain native browser functionality?

Reply:
Fixed